content/applications/finance/sign.rst

```diff
@@ -2,47 +2,57 @@
 Sign
 ====
 
-**Odoo Sign** allows you to send, sign and approve documents online.
+Odoo Sign allows you to send, sign and approve documents online, using electronic signatures.
 
-You can upload any PDF file and add drag-and-dropping fields on it. These fields are automatically
-filled out with the user's detail if they are logged in.
+An **electronic signature** shows a person’s agreement to the content of a document. Just like a
```

Comment:
nitpicking but: curly from copy-pase and it's not consistent with all quotes in the doc.

Reply:
also in the rest of the doc